### ISSUE TITLE
Bug when loading .ser file.

### ISSUE THREAD

**Issue by martinezmatias** [STAMP-project/test-runner#78]
Hi @monperrus @danglotb 

The last commit produces a bug when reading the serializer. https://github.com/STAMP-project/test-runner/commit/525e47947d29c9cde1be3a1a8c2873b4a51bddb7

In particular, the problem is that the the bytecode located at the resources (https://github.com/STAMP-project/test-runner/tree/master/src/main/resources/runner-classes/eu/stamp_project/testrunner) where not updated. Consequently, the runner has the old version of the PATH (with the "dspot" that was removed in the previous commit).

My questions are:

1) Why is necessary that the runner executes those classes copied from the resource folder (in the runner classpath I see the path to them ` test-runner/target/classes/runner-classes/`) and not the compiled classes (e.g., `test-runner/target/classes/runner-classes/`). I understand that the idea is to pass to the runner only the classes it needs and not all.

2) Is there an automated way to update `/resources/runner-classes/`? 

Thanks
Regards
Matias

**Comment by danglotb** [comment]
Hello @martinezmatias 

Thank you for raising this issue.

My answers to your questions: 

1. > I understand that the idea is to pass to the runner only the classes it needs and not all.
It is the reason. And if I remember well, it is more convenient to find theses classes from the resources folder when I used test runner as library.

2. there is a script, `./src/update-classes-in-resources.sh` that does the trick.

I opened #79 to update the distributed binaries.
